### ISSUE TITLE
Enquiry about the scroll backlog list. 

### ISSUE THREAD

**Issue by EABrown3** [uckelman/dw_op#43]
Hiya - issue raised to website-issues email address, from Esbiorn, 28 June 2019.

Service Desk (from david.cordes@gmail.com): is the scroll-backlog list going to end up on the new website?
Previously the list of backlog scrolls was accessible, presumably pulling from a databased that I would suspect was run by Signet, or, now I check the link, from part of the OP database (link used to be http://op.drachenwald.sca.org/backlog.html). Its possible that this content is now hosted elsewhere, predominantly for signets/scribes, but it is the sort of thing that I suspect non-scribes want to look at occasionally (if only to see if someone has started working on their scroll).

Esbiorn

**Comment by uckelman** [comment]
Yes, it will. The new backlog page even exists already, but there's not much point in displaying it at present since all the Signet data is stale. I'm waiting for Signet now for testing the data input form; once she's happy it's a few minutes work to get the backlog page back.
